src/plugins/workspace/public/components/service_card/workspace_list_card.tsx

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { Component } from 'react';
+import {
+  EuiPanel,
+  EuiLink,
+  EuiDescriptionList,
+  EuiIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSelect,
+  EuiButtonIcon,
+  EuiSpacer,
+  EuiListGroup,
+  EuiText,
+  EuiTitle,
+  EuiToolTip,
+  EuiEmptyPrompt,
+} from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+import { Subscription } from 'rxjs';
+import moment from 'moment';
+import _ from 'lodash';
```

Comment:
Prefer to only import what's needed, for example:
```typescript
import isEqual from 'lodash/isEqual'
```

Reply:
thanks, updated